messages/lightning.preview.app.md

```diff
@@ -71,74 +69,79 @@ Downloading
 
 # certificate.installation.description
 
-If you have not done so already, please install the self-signed certificate on your device before proceeding. The certificate file is located at
+Before proceeding, make sure you installed the self-signed certificate on your device. The certificate file is located at
 
 `%s`
 
-Follow the steps below to install the certificate:
+To install the certificate, follow these steps:
 
 %s
 
 # certificate.installation.steps.ios
 
-1. Drag and drop the file on to your booted simulator.
-2. Tap `Allow` to proceed with downloading the configuration file.
-3. Tap `Close` and navigate to `Settings > General > VPN & Device Management > localhost`.
-4. Tap `Install` in the title bar, in the warning window, and on the install button.
-5. In the `Profile Installed` view, confirm that the profile shows as `Verified` and Tap `Done`.
-6. Now navigate to `Settings > General > About > Certificate Trust Settings`.
-7. Toggle full trust for `localhost` to enable full trust.
-8. In the warning dialog, tap `Continue`.
+1. Drag and drop the file onto your booted simulator.
+2. Click `Allow` to proceed with downloading the configuration file.
```

Comment:
I don't know what the standard documenting convention is for mobile gestures like this, but the actual action will be a tap, not a click. But maybe users put more emphasis on the repeated designator in documentation, across device experiences? I don't know, so I figured I would ask.

Reply:
Oh but this will be in a simulator so it doesn't even apply. 🤦 Disregard!